### ISSUE TITLE
file.line no longer working after 2018.3.3 release

### ISSUE THREAD

**Issue by wildk1wi** [saltstack/salt#50319]
### Description of Issue/Question
Using line function from the file module to ensure the existence of a line in a Linux config file no longer works as expected. 

It is now inserting a new line every time the high state runs rather than just once.
As you can see this is now the contents of the /etc/yum.conf file.

[main]
cachedir=/var/cache/yum/$basearch/$releasever
keepcache=0
debuglevel=2
logfile=/var/log/yum.log
exactarch=1
obsoletes=1
gpgcheck=1
plugins=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
clean_requirements_on_remove=1
installonly_limit=5
bugtracker_url=http://bugs.centos.org/set_project.php?project_id=23&ref=http://bugs.centos.org/bug_report_page.php?category=yum
distroverpkg=centos-release

### Steps to Reproduce Issue
(Include debug logs if possible and relevant.)

As state file in a sls:
/etc/yum.conf:
  file.line:
    - content: clean_requirements_on_remove=1
    - after: plugins
    - indent: True
    - mode: ensure

Or from the command line:
salt-call file.line /etc/yum.conf content="clean_requirements_on_remove=1" after=plugins indent=true mode=ensure

Examples of the steps with working minion 2018.3.2 and with non working updated minion 2018.3.3

[2018-3-2-file.line.txt](https://github.com/saltstack/salt/files/2531431/2018-3-2-file.line.txt)
[2018-3-3-file.line.txt](https://github.com/saltstack/salt/files/2531432/2018-3-3-file.line.txt)

### Versions Report
(Provided by running `salt --versions-report`. Please also mention any differences in master/minion versions.)
[vagrant@ddw-firewall1 ~]$ sudo salt-call --versions-report
Salt Version:
           Salt: 2018.3.3
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: Not Installed
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.7.2
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.5.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.5 (default, Aug  4 2017, 00:39:18)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.3.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4
 
System Versions:
           dist: centos 7.4.1708 Core
         locale: UTF-8
        machine: x86_64
        release: 3.10.0-693.21.1.el7.x86_64
         system: Linux

**Comment by wildk1wi** [comment]
In the /etc/yum.conf case, as a work around I've moved the class to using the in ini module as the file conforms to the ini file spec.

**Comment by Ch3LL** [comment]
this is a duplicate of https://github.com/saltstack/salt/issues/50254
